content/blog/tech-innovation/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Tech innovation: Solving wildlife conservation's greatest challenge
```

Comment:
![image](https://user-images.githubusercontent.com/6032276/132096304-e1b93c90-d622-44ea-a7af-8104d0847faf.png)

```suggestion
title: "Tech innovation: Solving wildlife conservation's greatest challenge"
```

Reply:
This may also work.
```suggestion
title: >-
 Tech innovation: Solving wildlife conservation's greatest challenge
```